api/src/main/resources/messages.properties

```diff
@@ -77,3 +77,5 @@ idgen.addNewIdentifier = Add New Identifier...
 idgen.location = Location (optional)
 idgen.setupNewAutoGenerationOption=Add a new Auto Generation Option
 idgen.delete=Delete
+idgen.setAutoOrManualGenerationOption = Either automatic or manual generation must be enabled
```

Comment:
i introduced my error messages in the messages.properties file,on displaying them i was thinking of some thing like errors.rejectValue("value", "idgen.setAutoOrManualGenerationOption"); i tried this but it failed and it complained of the value attribute @ibacher